.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,55 @@
+name: Bug Report
+description: Create a bug report to help us fix issues with Quilt Standard Libraries.
```

Comment:
I would split this into 3 different templates:
1. I encountered a crash!
    - Focus on getting a crash report

2. Something is broken in game.
    - Focus on a description of the error

3. I found a bug with QSL.
    - Focus on the actual implementation issue from QSL
    
I'll treat this template as the first one.